robottelo/entities.py

```diff
@@ -1156,60 +1156,54 @@ class Meta(object):
         server_modes = ('sat')
 
     def create_missing(self, auth=None):
-        """Customize the process of auto-generating instance attributes.
+        """Create a bogus managed host.
+
+        The exact set of attributes that are required varies depending on e.g.
```

Comment:
I don't think ``e.g.`` is necessary here... try to read the same sentence w/o it...